.isort.cfg

```diff
@@ -3,6 +3,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = setuptools
 known_first_party = mmdet
-known_third_party = asynctest,cityscapesscripts,cv2,matplotlib,mmcv,numpy,onnx,pycocotools,robustness_eval,roi_align,roi_pool,seaborn,six,terminaltables,torch,torchvision
+known_third_party = PIL,asynctest,cityscapesscripts,cv2,matplotlib,mmcv,numpy,onnx,pycocotools,robustness_eval,roi_align,roi_pool,seaborn,six,terminaltables,torch,torchvision
```

Comment:
It seems that we usually do not use PIL in our codebase, may remove this.

Reply:
Should we remove it from runtime requirements as well? 
https://github.com/open-mmlab/mmdetection/blob/c40cafb1e1347645c1289634a40cc26b7b58163c/requirements/runtime.txt#L5

I committed using an environment generated using requirements files and pre-commit check forced me to add this to commit before it accepted the commit.

Reply:
No, we cannot remove it from the requirements because of the potential incompatibility of torchvision and PILLOW versions.

Reply:
But the mmdet code indeed does not use PIL, it is wierd.

Reply:
I have removed PIL from .isort.cfg.